app/bundles/EmailBundle/Translations/en_US/messages.ini

```
@@ -296,6 +297,7 @@ mautic.email.sent.emails="Sent emails"
 mautic.email.sent="Email sent"
 mautic.email.stage.action.open="Opens an email"
 mautic.email.stage.action.send="Sends an email"
+mautic.email.stats.options.title="E-mails"
```

```suggestion
mautic.email.stats.options.title="Emails"
```
Let's use Emails without the dash for consistency with the rest of the translations.